code/modules/clothing/clothing.dm

```diff
@@ -438,24 +438,27 @@ BLIND     // can't see anything
 	name = "suit"
 	var/fire_resist = T0C+100
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS|LEGS
-	allowed = list(/obj/item/weapon/tank/emergency_oxygen)
+	allowed = list(
+		/obj/item/weapon/storage/pouch/,
+		/obj/item/weapon/gun,
+		/obj/item/weapon/melee,
+		/obj/item/weapon/material,
+		/obj/item/ammo_magazine,
+		/obj/item/ammo_casing,
+		/obj/item/weapon/handcuffs,
+		/obj/item/weapon/tank,
+		/obj/item/device/suit_cooling_unit,
+		/obj/item/weapon/cell,
+		/obj/item/weapon/storage/fancy/cigarettes,
+		/obj/item/weapon/flame/lighter,
+		/obj/item/device/lighting/toggleable/flashlight,
```

Comment:
I'm thinking these three entries could be reduced to 
/obj/item/weapon/storage/fancy
/obj/item/weapon/flame
/obj/item/device/lighting

Any thoughts on wierdness that might result? anything in those types that you think shouldn't fit there?